homeassistant/components/persistent_notification/services.yaml

```diff
@@ -33,3 +33,7 @@ dismiss:
       example: 1234
       selector:
         text:
+
+dismiss_all:
+  name: Dismiss All
+  description: Remove all notifications from the frontend.
```

Comment:
Let’s not mention frontend here since this api doesn’t operate on the frontend. The frontend just happens to consume it